<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectView</class>
 <widget class="QWidget" name="ConnectView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Enter Nintendo Switch details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP Address:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_address"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_port">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_info">
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Not connected.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_connect">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>released()</signal>
   <receiver>ConnectView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
